docs/grpc_disruption.md

```diff
@@ -0,0 +1,96 @@
+# GRPC disruption
+
+The `grpc` field offers a way to inject spoofed gRPC responses on the server-side. To get this disruption to work, you must apply some code changes to the instantiation of your gRPC server.
+
+* `port` is the port exposed on target pods (the target pods are specified in `spec.selector`)
+* `endpoints` is a list of endpoints to alter (a spoof configuration is referred to as an `alteration`)
+  * `<endpoints[i]>.endpoint` indicates the fully qualified api endpoint to override (ex: `/<package>.<service>/<method>`)
+  * Exactly one of `<endpoints[i]>.error` or `<endpoints[i]>.override` should be defined per endpoint alteration, and the only override currently supported is `{}` which returns `emptypb.Empty`
+  * `<endpoints[i]>.query_pct` defines (out of 100) how frequently this alteration should occur; you may have multiple alterations per endpoint, but you cannot specify a sum total of more than 100 percent for any given endpoint
+
+Note: At this time, Chaos Controller does not support:
+* disrupting gRPC Streams
+* chaining the disruption interceptor on another interceptor you already have
```

Comment:
while we haven't tested this, we aren't exactly sure it doesn't work either, so maybe mention this is more of a "caveat emptor" sort of unsupported, rather than a "definitely does not work" unsupported like grpc streams